app/models/stat.rb

```diff
@@ -75,7 +83,11 @@ def agents_sample
 
   def all_organisations
     @all_organisations ||=
-      department.nil? ? Organisation.all : department.organisations
+      if statable_type == "Organisation"
+        Organisation.where(id: statable_id)
+      else
+        department.nil? ? Organisation.all : department.organisations
+      end
   end
```

Comment:
Si tu n'as pas besoin d'itérer sur une collection ActiveRecord tu peux peut-être faire:
```ruby
statable_type == "Organisation" ? [organisation] : statable.organisations
```